sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_base_test.hpp

```diff
@@ -122,4 +121,16 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
     }
   };
 
+  namespace {
+    static std::string GetSuffix(const testing::TestParamInfo<int>& info)
+    {
+      return std::to_string(info.param);
```

Comment:
what about adding a little more description here like `"KeySize-"+std::to_string(info.param)` or something which helps reading the console out better.

Reply:
param is key size , the name of the test becomes "testname/13" or testname/2048 so it's test with the associated key size